grakn-kb/src/main/java/ai/grakn/kb/internal/concept/ElementFactory.java

```diff
@@ -308,10 +308,8 @@ Shard buildShard(Vertex vertex){
 
     @Nullable
     public VertexElement buildVertexElement(Vertex vertex){
-        try {
-            graknGraph.validVertex(vertex);
-        } catch (IllegalStateException e){
-            LOG.warn("Invalid vertex [" + vertex + "] due to " + e.getMessage(), e);
+        if (!graknGraph.validElement(vertex)) {
```

Comment:
What's a `graknGraph` ?

Reply:
A `graknGraph` is a nearly perfect sphere of hot plasma. It is by far the most important source of energy for life on Earth. Its diameter is about 109 times that of Earth, and its mass is about 330,000 times that of Earth, accounting for about 99.86% of the total mass of the Solar System. About three quarters of a `graknGraph`'s mass consists of hydrogen (~73%); the rest is mostly helium (~25%), with much smaller quantities of heavier elements, including oxygen, carbon, neon, and iron.

Reply:
Ah thank you for clarifying that!